asmcomp/cmm.mli

```diff
@@ -134,10 +134,10 @@ and expression =
   | Clet of Ident.t * expression * expression
   | Cassign of Ident.t * expression
   | Ctuple of expression list
-  | Cop of operation * expression list
+  | Cop of operation * expression list * Debuginfo.t
   | Csequence of expression * expression
   | Cifthenelse of expression * expression * expression
-  | Cswitch of expression * int array * expression array
+  | Cswitch of expression * int array * expression array * Debuginfo.t
```

Comment:
It probably should -- see reply to @alainfrisch .  I think it was just that I'd empirically noticed that switches seemed to be worse (perhaps because the scrutinee is more often just a variable?).